### PR TITLE
[MB-1598] update interface with properties, monodevelop style

### PR DESCRIPTION
**Pull request by marc-scig**
Just trying to make the plugin interface more palatable to the .NET world. This means rewriting things to use properties where possible (instead of argless setters and getters), and trying to adopt some consistent style guidelines. FWIW MonoDevelop has its own style guide derived from Mono, which explains some of the odd seeming choices the editor makes for you by default 

http://www.mono-project.com/community/contributing/coding-guidelines/

### CODE REVIEW

File: Assets/Scripts/UrbanAirship.cs [hunk 1]
```diff
@@ -1,58 +1,53 @@
-﻿/*
- Copyright 2015 Urban Airship and Contributors
```
Comment (rlepinski) [inline Assets/Scripts/UrbanAirship.cs]: Why did you remove the copyright?

File: Assets/UrbanAirship/Platforms/Android/UAirshipPluginAndroid.cs [hunk 2]
```diff
@@ -14,128 +14,119 @@ public class UAirshipPluginAndroid : IUAirshipPlugin
 {
```
Comment (rlepinski) [inline Assets/UrbanAirship/Platforms/Android/UAirshipPluginAndroid.cs]: We should probably namespace the platform files

File: Assets/Scripts/UrbanAirship.cs (hunk 1 above)
Comment (marc-scig) [inline Assets/Scripts/UrbanAirship.cs]: Hm, that was not on purpose

File: Assets/UrbanAirship/Platforms/Android/UAirshipPluginAndroid.cs (hunk 2 above)
Comment (marc-scig) [inline Assets/UrbanAirship/Platforms/Android/UAirshipPluginAndroid.cs]: good call

**Comment by rlepinski** [comment]
:+1:
